src/pages/patientView/mutation/column/MrnaExprColumnFormatter.tsx

```diff
@@ -0,0 +1,110 @@
+                    <span><b>Percentile: </b>{exprData.percentile}</span><br/>
+                </div>
+            );
+        } else {
+            return (<span>mRNA data is not available for this gene.</span>);
+
+        }
+    }
+
+    private static getTdContents(data:IColumnFormatterData<MutationTableRowData>, mrnaExprData: any) {
+        const barWidth = 30;
+        const circleRadius = 3;
+        const barXLeft = -circleRadius;
+        const barXRight = barWidth + circleRadius;
+        const exprData = MrnaExprColumnFormatter.getData(data, mrnaExprData);
+        if (exprData) {
+            return (<svg
+                width={66}
+                height={12}
+            >
+                <text x={66} y={11} textAnchor="end" fontSize={10}>
+                    {Math.round(exprData.percentile)+"%"}
+                </text>
+                <g transform="translate(3,0)">
+                    <line x1={barXLeft}
+                          y1={6}
+                          x2={barXRight}
+                          y2={6}
+                          style={{stroke:"gray", strokeWidth:2}}
+                    />
+                    <circle cx={MrnaExprColumnFormatter.getCircleX(data, exprData.percentile, barWidth, circleRadius)}
+                            cy={6}
+                            r={circleRadius}
+                            fill={MrnaExprColumnFormatter.getCircleFill(exprData.percentile)}
+                    />
+                </g>
+            </svg>);
+        } else {
+            return (
+                <span style={{color: "gray", fontSize:"xx-small", textAlign:"center"}}
+                      alt="mRNA data is not available for this gene."
+                >
+                    NA
+                </span>
+            );
+        }
+    }
+
+    private static getData(data: IColumnFormatterData<MutationTableRowData>, mrnaExprData:any) {
+        if (!data.rowData || data.rowData.length === 0) {
+            return null;
+        }
+        const sampleId = data.rowData[0].sampleId;
+        const entrezGeneId = data.rowData[0].entrezGeneId;
+        const ret = mrnaExprData && mrnaExprData[sampleId] && mrnaExprData[sampleId][entrezGeneId];
+        return ret || null;
+    }
+
+    public static renderFunction(data: IColumnFormatterData<MutationTableRowData>, columnProps: any) {
+        const exprData = MrnaExprColumnFormatter.getData(data, columnProps.data);
+        return (<Td key={data.name} column={data.name} value={exprData? exprData.percentile : Number.POSITIVE_INFINITY}>
+            <Tooltip
+                placement="left"
+                overlay={MrnaExprColumnFormatter.getTooltipContents(data, columnProps.data)}
+                arrowContent={<div className="rc-tooltip-arrow-inner"/>}
+            >
+                {MrnaExprColumnFormatter.getTdContents(data, columnProps.data)}
+            </Tooltip>
+        </Td>);
+    }
+    public static sortFunction(a:number, b:number):number {
```

Comment:
Do we need a custom sort function for this column? It may work fine by default, since the Td value is a number.